tools/trace2syz/trace2syz/util.go

```diff
@@ -0,0 +1,857 @@
+			"KEYCTL_CLEAR":                "$clear",
+			"KEYCTL_LINK":                 "$link",
+			"KEYCTL_UNLINK":               "$unlink",
+			"KEYCTL_SEARCH":               "$search",
+			"KEYCTL_READ":                 "$read",
+			"KEYCTL_CHOWN":                "$chown",
+			"KEYCTL_SETPERM":              "$setperm",
+			"KEYCTL_INSTANTIATE":          "$instantiate",
+			"KEYCTL_NEGATE":               "$negate",
+			"KEYCTL_SET_REQKEY_KEYRING":   "$set_reqkey_keyring",
+			"KEYCTL_SET_TIMEOUT":          "$set_timeout",
+			"KEYCTL_ASSUME_AUTHORITY":     "$assume_authority",
+			"KEYCTL_GET_SECURITY":         "$get_security",
+			"KEYCTL_SESSION_TO_PARENT":    "$session_to_parent",
+			"KEYCTL_REJECT":               "$reject",
+			"KEYCTL_INSTANTIATE_IOV":      "$instantiate_iov",
+			"KEYCTL_INVALIDATE":           "$invalidate",
+			"KEYCTL_GET_PERSISTENT":       "$get_persistent",
+		}
+
+
+		prctlLabels = map[string]string{
+			"PR_GET_DUMPABLE":             "$void",
+			"PR_GET_KEEPCAPS":             "$void",
+			"PR_GET_NO_NEW_PRIVS":         "$void",
+			"PR_GET_SECCOMP":              "$void",
+			"PR_GET_SECUREBITS":           "$void",
+			"PR_GET_TIMERSLACK":           "$void",
+			"PR_GET_TIMING":               "$void",
+			"PR_TASK_PERF_EVENTS_DISABLE": "$void",
+			"PR_TASK_PERF_EVENTS_ENABLE":  "$void",
+			"PR_MCE_KILL_GE":              "$void",
+			"PR_CAPBSET_READ":             "$intptr",
+			"PR_CAPBSET_DROP":             "$intptr",
+			"PR_SET_CHILD_SUBREAPER":      "$intptr",
+			"PR_SET_DUMPABLE":             "$intptr",
+			"PR_SET_FPEMU":                "$intptr",
+			"PR_SET_KEEPCAPS":             "$intptr",
+			"PR_SET_NO_NEW_PRIVS":         "$intptr",
+			"PR_SET_PDEATHSIG":            "$intptr",
+			"PR_SET_SECUREBITS":           "$intptr",
+			"PR_SET_TIMERSLACK":           "$intptr",
+			"PR_SET_TIMING":               "$intptr",
+			"PR_SET_TSC":                  "$intptr",
+			"PR_SET_UNALIGN":              "$intptr",
+			"PR_MCE_KILL":                 "$intptr",
+			"PR_GET_CHILD_SUBREAPER":      "$getreaper",
+			"PR_GET_ENDIAN":               "$getreaper",
+			"PR_GET_FPEMU":                "$getreaper",
+			"PR_GET_FPEXC":                "$getreaper",
+			"PR_GET_PDEATHSIG":            "$getreaper",
+			"PR_GET_TID_ADDRESS":          "$getreaper",
+			"PR_GET_TSC":                  "$getreaper",
+			"PR_GET_UNALIGN":              "$getreaper",
+			"PR_SET_ENDIAN":               "$setendian",
+			"PR_SET_FPEXC":                "$setfpexc",
+			"PR_SET_NAME":                 "$setname",
+			"PR_GET_NAME":                 "$getname",
+			"PR_SET_PTRACER":              "$setptracer",
+		}
+	*/
+	specialConsts = map[string]uint64{
+		"_LINUX_CAPABILITY_VERSION_1":                         0x19980330,
+		"_LINUX_CAPABILITY_VERSION_2":                         0x20071026,
+		"_LINUX_CAPABILITY_VERSION_3":                         0x20080522,
+		"PROT_NONE":                                           0,
```

Comment:
We already have all these consts in prog.Target

Reply:
yeah this is a good example of maintainability. I added these before syzkaller added these constants

Reply:
i will remove them now

Reply:
So ```PROT_NONE``` is definitely redundant but it seems that ```_LINUX_CAPABILITY_VERSION_3``` is defined as ```cap_version = 0x19980330, 0x20071026, 0x20080522`` in Syzkaller`.It seems there are two cases:

1. Syzkaller names a constant differently than Strace. 
2. Strace contains constant that Syzkaller doesn't recognize. 

In the second case it is simple: we just add the constant. What should we do about the first case?

Reply:
We can change syzkaller descriptions to something like:
```
cap_version = _LINUX_CAPABILITY_VERSION_1, _LINUX_CAPABILITY_VERSION_2, _LINUX_CAPABILITY_VERSION_3
define _LINUX_CAPABILITY_VERSION_1 0x19980330
define _LINUX_CAPABILITY_VERSION_2 0x20071026
define _LINUX_CAPABILITY_VERSION_3 0x20080522
```